src/vivarium_census_prl_synth_pop/results_processing/formatter.py

```diff
@@ -92,6 +92,22 @@ def format_age(data: pd.DataFrame) -> pd.Series:
     return data["age"].astype(int)
 
 
+def format_copy_age(data: pd.DataFrame) -> pd.Series:
+    no_copy_idx = data.index[data["copy_age"].isna()]
+    column = data["copy_age"].astype(str)
+    column = column.apply(lambda row: row.split(".")[0])
+    column.loc[no_copy_idx] = np.nan
+    return column
+
+
+def format_copy_ssn(data: pd.DataFrame) -> pd.Series:
+    no_copy_idx = data.index[data["copy_ssn"].isna()]
+    column = data["random_seed"].astype(str) + "_" + data["copy_ssn"].astype(str)
+    column = column.apply(lambda row: row[:-2])
```

Comment:
Wht are the last two characters that you're not including?

Reply:
".0". Because of nans, the simulant ids in copy_ssn are 1234.0 instead of 1234.